src/coreclr/vm/commodule.cpp

```diff
@@ -860,7 +860,7 @@ void QCALLTYPE COMModule::GetFullyQualifiedName(QCall::ModuleHandle pModule, QCa
         if (*fileName != 0) {
                 retString.Set(fileName);
         } else {
-            retString.Set(W("<Unknown>"));
+            retString.Set(W("<unknown>"));
```

Comment:
What is this trying to be consistent with? There is a lot of code out there that expects the current casing and that will break with this change. For example:

https://github.com/microsoft/perfview/blob/6b4771eecba5ed00555a34d70e19a8742af4ee0a/src/TraceEvent/NativeDlls.cs#L29

Also, the current casing is documented in the docs.

Reply:
Ugh, I had hoped this would be uncommon.

The case I found was in `RoModule.cs` as part of MetadataLoadContext. It uses lowercase `<unknown>`.

I'll revert the changes to `<Unknown>` for now, but I think MLC is inconsistent, so anyone relying on the behavior being consistent would be surprised. Maybe that's fine because the reflection APIs and MLC APIs don't cross over much.

Reply:
I think it would be fine to fix MetadataLoadContext to use `<Unknown>`. I think it is unlikely that anybody depends on the exact MetadataLoadContext behavior in this case, so it should be fine to treat it as a simple MetadataLoadContext bugfix.